packages/lesswrong/components/ea-forum/onboarding/EAOnboardingStage.tsx

```diff
@@ -102,8 +109,62 @@ const styles = (theme: ThemeType) => ({
   spinner: {
     height: "20px !important",
   },
+  closeButtonOuter: {
+    position: "absolute",
+    top: 16,
+    right: 24,
+  },
+  close: {
+    cursor: "pointer",
+  },
+  logoutDialog: {
+    zIndex: theme.zIndexes.confirmLogoutModal,
+    minWidth: 300,
+  },
+  logoutDialogTitle: {
+    marginTop: 0,
+  },
+  logoutDialogText: {
+    marginTop: 0,
+    marginBottom: 2,
+  },
+  logoutDialogActions: {
+    marginRight: 16,
+    marginBottom: 16,
+  },
 });
 
+const LogoutConfirmationDialog = (
+  {open, onClose, classes}:
+  {open: boolean, onClose: () => void, classes: ClassesType<typeof styles>},
+) => {
+  const currentUser = useCurrentUser();
+  
+  const {EAButton, LWDialog, Typography} = Components;
+  return <LWDialog open={open} onClose={onClose} className={classes.logoutDialog}>
+    <DialogTitle disableTypography>
+      <Typography variant="display1" className={classes.logoutDialogTitle}>
+        Confirm Logout
+      </Typography>
+    </DialogTitle>
+    <DialogContent>
+      <DialogContentText>
+        <p className={classes.logoutDialogText}>You are currently logged in with the email</p>
```

Comment:
This creates a `<p> cannot appear as a descendant of <p>` warning in the console, could you put this all directly in the `DialogContentText`:
```suggestion
    <DialogContentText className={classes.logoutDialogText}>
        You are currently logged in with the email {currentUser?.email ?? "(no email found)"}, but
        have not chosen a username.
      </DialogContentText>
```
You'll need to tweak some other things, like adding a max-width to the dialog, as this was previously handled by having the 3 separate `<p>`s

Reply:
Thank you! I played around with it before and wrapping with long emails looked pretty bad, so I opted for three lines. But I've fixed the p descendent of p issue.